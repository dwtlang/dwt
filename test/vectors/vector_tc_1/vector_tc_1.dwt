var v = { 1, 2, 3 }

print v[2]

v[100] = 543

print v[100]
print v[99]
