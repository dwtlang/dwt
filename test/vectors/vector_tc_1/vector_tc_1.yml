---
description:      "basic vector operations"
name:             vector_tc_1
src:              vector_tc_1.dwt
out:              vector_tc_1.out
err:              vector_tc_1.err
exitcode:         0
loop:             1
skip:             false
