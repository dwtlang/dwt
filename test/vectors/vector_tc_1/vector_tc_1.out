3
543
<nil>
